### PR TITLE
New DevTools Digest

### PR DESCRIPTION
**Pull request by pbakaus**
This adds a new DevTools Digest to the update section. Please review! Would do me a great favor if you could launch this ASAP.

@petele @gauntface @Meggin @jpmedley

### CODE REVIEW

**Comment by gauntface** [comment]
LGTM form me in terms of markup

**Comment by addyosmani** [comment]
One super minor comment on content but otherwise LGTM2

**Comment by jpmedley** [comment]
DevTools is Kayce. I'm adding him to the discussion.
@kaycebasques

**Comment by pbakaus** [comment]
@addyosmani thanks, good to know. We've heard a few of these complains. We will address it in some way in a next update. Sad we can't go full throttle as originally planned. @paulirish

**Comment by kaycebasques** [comment]
@pbakaus reviewing when I get out of my team meeting at 10:30am

**Comment by pbakaus** [comment]
Doing some last minute changes in the next half hour or so, please do not merge yet.

**Comment by pbakaus** [comment]
OK all good now! Review and merge at your convenience.

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+featured_image: /web/updates/images/2015/11/devtools-digest/animations.png
+---
+
+<p class="intro">Learn how DevTools is going mobile first with a new, streamlined Device Mode that’s always on. Use the color buttons to quickly add colors to your selectors and find out what’s coming to DevTools soon.</p>
+
+## A glimpse into the future of authoring
+
+{% ytvideo dJR-n8szgBc %}
+
+We’re just coming back from the Chrome Dev Summit where I showed you what working with DevTools looks like today and in the future. I usually don’t mention any features that are still experiments or heavy work of progress in this digest but I’m making an exception this time. If you don’t have time to watch the whole talk, here’s the gist:
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: heavy **works** of progress

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Device Mode v2
+
+{% ytvideo r-uc-fENa7w %}
+
+We’re still heavily working on this new, bold iteration of the **Device Mode** but wanted to give everyone an opportunity to try it out, so we’ve enabled it in your Canary today. With the changes, we are pushing DevTools into a mobile-first future where mobile development is the default, and Desktop development is the “add-on”. Expect more iteration over the next few weeks with an extended blog post when we’re done.
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: s/the Device Mode/Device Mode/

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Powerful Animation Inspection
+
+{% ytvideo MjaYjUram14 %}
+
+The work-in-progress **Animation Inspection** gives you a full, detailed picture over what’s happening on anything moving. Instead of showing you a transition on one element at a time, we added heuristics that group complex animations to so you can focus on all you’re seeing. Have a look at the video and like with Device Mode, we’ll offer a bigger, standalone blog post when we’re fully launched.
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: s/animations to so/animations so/

Comment (kaycebasques) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: Have a look at the video. We'll offer a bigger, standalone...

**Comment by kaycebasques** [comment]
LGTM :+1: 

Next digest, if you'd like, send me a Google Doc draft and I'll show you a bunch of tricks for making the writing sharper and more concise, while still maintaining your friendly and entertaining writing voice.

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown (hunk 2 above)
Comment (pbakaus) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: I kinda like the "the" there, even though it is named.

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown (hunk 1 above)
Comment (pbakaus) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: fixing

File: src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown (hunk 3 above)
Comment (pbakaus) [inline src/content/en/updates/posts/2015/11/devtools-digest-cds-edition.markdown]: fixing

**Comment by pbakaus** [comment]
Cool, addressed your comments. Will give you a heads up on a doc next time!

**Pull request merged**